documenten/techniek/designdecisions.md

```diff
@@ -14,4 +14,13 @@ We gebruiken NLX voor elke call waarbij vertrouwelijke gegevens worden opgevraag
 We hanteren semantic versioning. 
 We hanteren https://docs.geostandaarden.nl/api/API-Strategie/#versioning  als uitgangspunt voor versiebeheer
 
+Timetravel
+-------
+A part of the [haal centraal](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/master/api-specificatie/Bevraging-Ingeschreven-Persoon/openapi.yaml) the concept of timetravel has been introduced, as in getting the version of an object as it was on a given date. For this the `geldigop` [see the docs](file:///C:/Users/ruben/Desktop/doc_gba_historie.html#operation/getBewoningen) header is used. In addition the `geldigvan` and `geldigtot` are introduced as collection filters. 
 
+The commonground proto componant natively supports time traveling on all resources that are annotaded with the @Gedmo\Loggable, this is done by adding the ?validOn=[date] query to a request, date can either be a datetime or datedatime string. Any value supported by php's [strtotime()](https://www.php.net/manual/en/function.strtotime.php) funtion is supported. Keep in mind that this returns the entity a as it was valid on that time or better put, the last changed version BEFORE that moment. To get a complete list of all changes on an item the [/audittrail](#Audittrail
```

Comment:
strtotime is language specific, let's replace it with something like 'Any valid iso8601 datetime'

Reply:
Eens, de werking zou taalonafhankelijk beschreven moeten zijn. Ik zou ook het stuk tekst over het proto component er uit halen.

Reply:
Eens met bijde, begin moet zijn  "Our components ..." en dan iso8601